sprint/faq.md

```diff
@@ -0,0 +1,2 @@
+# Frequently Asked Questions
```

Comment:
@reshamas do you have some questions we should add here?

Reply:
We have not received any questions yet.

Reply:
Then it is probably best we hide that page until we have something to add there. @symeneses can you remove it from the toctree and add 

```
---
orphan: true
---
```

at the top of the file?

Reply:
Done!